### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by Vathsan**

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -36,6 +36,20 @@
         <dependency>
             <groupId>org.wso2.ei</groupId>
             <artifactId>org.wso2.micro.integrator.server</artifactId>
+            <exclusions>
```
Comment (sdkottegoda) [inline distribution/pom.xml]: Let's remove these from org.wso2.micro.integrator.server.feature itself

Comment (Vathsan) [reply distribution/pom.xml]: Done in 838be769aaa067edf0f4aa07c78faa6d3dd200ba

**Comment by Nirothipan** [comment]
Let's merge this after stabilizing integration tests

**Comment by Nirothipan** [comment]
Closing this as we have to update individual components and upgrade versions.

**Pull request closed without merging**